content/en/getting_started/synthetics/browser_test.md

```diff
@@ -57,7 +63,7 @@ For example, to record test steps that map a user's journey from adding an item
 1. Navigate to one of the furniture sections, for instance **Chairs**, and select **Add to cart**.
 2. Click on **Cart**, click **Checkout**.
 3. Add the **assertion** “Test text is present on the active page” to confirm the words “Thank you” are on the page.
-    **Note**: Your last browser test step must be an **assertion**. This will ensure your test ended up on the expected page and found the expected element.
+   **Note**: Your last browser test step must be an **assertion**. This will ensure your test ended up on the expected page and found the expected element.
```

Comment:
```suggestion
   **Note**: Your final browser test step must be an **assertion**. This will ensure your test ended up on an expected page and found the expected element.
```